pkg/cmd/openshift-sdn/cmd.go

```diff
@@ -174,11 +188,10 @@ func (sdn *OpenShiftSDN) ValidateAndParse() error {
 		return err
 	}
 
-	cniConfDir := "/etc/cni/net.d"
-	if val, ok := sdn.NodeConfig.KubeletArguments["cni-conf-dir"]; ok && len(val) == 1 {
-		cniConfDir = val[0]
```

Comment:
@squeed so we're dropping the enforced same CNI configdir path between kubelet and the SDN and just making sure we manage that manually through the daemonset YAML now? Which is fine, just want to be sure.

Reply:
basically, yes - given that NodeConfig doesn't exist anyways.